### PR TITLE
fixes some amtag bugs

### LINKED ISSUE #19015: new action buttons make it so you cant use slabs in hand
### Severity

Inconvenience

### Short Description

The name is the description. Normally, when you're a clockie with a clockwork slab, you can bind spells to an action button making it way easier to use. The new action button update breaks this, as whenever you put the clockwork slab in your hand, they all vanish, yet they all go back when you put it elsewhere.

### Reproduction

1. Be cogger
2. Have slab
3. Move slab to hand
4. Spell action buttons all vanish

### Round ID

47359

### Test Merges

n/a

### BYOND Username

Yarinoi

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

_No response_

### LINKED ISSUE #19018: Infinite screenshake
### Severity

Exploit

### Short Description

Mushroom hallucinogen wears off 30 minutes after metabolization.

### Reproduction

1. Make mushroom hallucinogen + gaseous decomposition plant.
2. Throw at self.
3. Suffer.

### Round ID

Round ID: 47362

### Test Merges

None!

### BYOND Username

kirbykraze2

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

May or may not be permanent, an admin helped remove it.

### LINKED ISSUE #19019: new button system broke jumpboots implant
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: Basically the title, it goes on cooldown and does nothing

 Submitted by: PibeCantina (341102461504716800)

### LINKED ISSUE #19020: New UI broke diagnostic hud? (partially
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: Diagnostic hud cannot see nanite level anymore (it still works on everything else)

 Submitted by: PibeCantina (341102461504716800)

### LINKED ISSUE #19022: Clock Cultist Watch Bugs when held, Visor button did not work, Kindle is spamable.
## Test Merges: 
Not Supplied / None
## Reproduction:
Description:  After trying to use clock cultists abilities, holding the Clock Watch, makes all the quickbound spells disappear off the top of the screen.  The Judicial Visor I also could not use at all, plus the new cursor after using the Healing spell stuck on my mouse cursor.  Also someone was able to spam kindle no cooldown like a machine gun.

 Submitted by: Tawaporah Tenderpaw (241348083907166219)

### LINKED ISSUE #19027: rust heretic mostly broken
### Severity

Round Breaking

### Short Description

Heretics have most of their abilities, and all ways of transmutating things are broken

### Reproduction

1. Be a rust heretic
2. Make a rune
3. Try to choose your target with your heart.  "Your ritual failed! You used the wrong components or are missing something important! x4"
4. Try to use your hand on floor, wall, door or a borg.
5. nothing happens
6. next. Try to create an item, with the the things researched example: Heretic armor or heretic flask. with the correct things on the rune
7. "Your ritual failed! You used the wrong components or are missing something important! x4"
8. And finally get perma'd because you can't do anything
9. Cry to admins

### Round ID

Round ID: 47376

### Test Merges

_No response_

### BYOND Username

Kmla

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

this was caused by #18514. using your hand on a person or drawing a rune, gathering essence from a rift or researching stuff still works though

### LINKED ISSUE #19028: Heretic transmutation rune is completely non-functional
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: non of the crafting works and when you try to find a target with the heart it thinks that you already have a target when you dont yet

 Submitted by: Master Drake (598557108551811073)

### LINKED ISSUE #19045: Sentient Disease is broken
### Severity

Round Breaking

### Short Description

Voluntary coughing doesnt work as sentient disease

### Reproduction

1. Become sentient disease per midround
2. Press the button
3. Host doesnt cough

### Round ID

Round #47388

### Test Merges

None

### BYOND Username

kondafairydragon

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

_No response_

### LINKED ISSUE #19049: Bloodsucker round, Event Manager constantly spawning more as random events
## Test Merges: 
Not Supplied / None
## Reproduction:
Description: It happening once or twice is one thing. But I saw it happen during this round no less than 5 times before the server crashed.

 Submitted by: Aquizit (168642050915434496)

### PR DESCRIPTION
**Pull request by tattax**
# Document the changes in your pull request

some things work again

temporarily removes observe button cause i was solving that but we have bigger problems

closes: https://github.com/yogstation13/Yogstation/issues/19028
closes: https://github.com/yogstation13/Yogstation/issues/19022
closes: https://github.com/yogstation13/Yogstation/issues/19027
closes: https://github.com/yogstation13/Yogstation/issues/19019
closes: https://github.com/yogstation13/Yogstation/issues/19018
closes: https://github.com/yogstation13/Yogstation/issues/19015
closes: https://github.com/yogstation13/Yogstation/issues/19049
closes: https://github.com/yogstation13/Yogstation/issues/19045
closes: https://github.com/yogstation13/Yogstation/issues/19020

also cleans up the internals icon that was supposed to be removed on the original pr
:cl:  
bugfix: fixed some antag bugs (i.e clockcult and heretic)
bugfix: fixed some action bugs (i.e jumpboots and quickbinds) + correct a typing mistakes that caused a status effect bug
/:cl:

### CODE REVIEW

File: code/__DEFINES/hud.dm [hunk 1]
```diff
@@ -124,8 +124,8 @@
 //Middle right (status indicators)
 #define ui_healthdoll "EAST-1:28,CENTER-2:13"
 #define ui_health "EAST-1:28,CENTER-1:15"
-#define ui_internal "EAST-1:28,CENTER:17"
-#define ui_mood "EAST-1:28,CENTER-4:10"
+#define ui_internal "EAST-1:28,CENTER+1:17"
+#define ui_mood "EAST-1:28,CENTER:17"
```
Comment (tattax) [inline code/__DEFINES/hud.dm:128]: moves mood up since we cleaned space

File: code/__byond_version_compat.dm [hunk 2]
```diff
@@ -31,19 +31,37 @@
...
-// So we want to have compile time guarantees these procs exist on local type, unfortunately 515 killed the .proc/procname syntax so we have to use nameof()
+// So we want to have compile time guarantees these methods exist on local type, unfortunately 515 killed the .proc/procname and .verb/verbname syntax so we have to use nameof()
+// For the record: GLOBAL_VERB_REF would be useless as verbs can't be global.
+
 #if DM_VERSION < 515
+
 /// Call by name proc reference, checks if the proc exists on this type or as a global proc
 #define PROC_REF(X) (.proc/##X)
+/// Call by name verb references, checks if the verb exists on either this type or as a global verb.
+#define VERB_REF(X) (.verb/##X)
```
Comment (tattax) [inline code/__byond_version_compat.dm:42]: needed for 515 but wasn't ported before

File: code/datums/actions/items/clockcult.dm [hunk 3]
```diff
@@ -8,19 +8,6 @@
 		return FALSE
 	return ..()
 
-/datum/action/item_action/clock/toggle_visor
```
Comment (tattax) [inline code/datums/actions/items/clockcult.dm:11]: remove this in favor of the cooldown one

File: code/game/data_huds.dm [hunk 4]
```diff
@@ -18,7 +18,7 @@
 /datum/atom_hud/data
 
 /datum/atom_hud/data/human/medical
-	hud_icons = list(STATUS_HUD, HEALTH_HUD)
+	hud_icons = list(STATUS_HUD, HEALTH_HUD, NANITE_HUD)
```
Comment (tattax) [inline code/game/data_huds.dm:21]: despite me putting this here i don't know if it's fixed so i didn't ping the issue

File: code/modules/antagonists/clockcult/clock_items/clockwork_slab.dm [hunk 5]
```diff
@@ -98,24 +98,27 @@
...
 	. = list()
 	if(isinhands && item_state && inhand_overlay)
 		var/mutable_appearance/M = mutable_appearance(icon_file, "slab_[inhand_overlay]")
 		. += M
 
-/obj/item/clockwork/slab/proc/check_on_mob(mob/user)
-	if(user && !(src in user.held_items) && slab_ability?.owner) //if we happen to check and we AREN'T in user's hands, remove whatever ability we have
+/obj/item/clockwork/slab/proc/check_on_mob(mob/user, slot)
+	if(!user)
+		CRASH("No user on dropped slab.")
```
Comment (tattax) [inline code/modules/antagonists/clockcult/clock_items/clockwork_slab.dm:117]: for better logging but if we see some edge cases i'll remove in a future pr

**Review by ToasterBiome — APPROVED** [review]
yeah this is totally not gonna break more shit

**Pull request merged**